src/platforms/android/index.mdx

```diff
@@ -764,3 +764,13 @@ Example of uploading all your .so files:
 sentry-cli login
 sentry-cli upload-dif -o {YOUR ORGANISATION} -p {PROJECT} build/intermediates/merged_native_libs/{buildVariant}
 ```
+
+Due to current limitations in how `sentry-native` works, it cannot yet identify native libraries loaded directly from `.apk` files.
```

Comment:
I'd still move the whole section right after the title `Integrating the NDK`

this should be the first step to be read.